sdk/storage/storage-blob/src/generated/src/storageClientContext.ts

```diff
@@ -39,7 +39,7 @@ export class StorageClientContext extends coreHttp.ServiceClient {
 
     super(undefined, options);
 
-    this.version = '2019-12-12';
+    this.version = '2019-10-10';
```

Comment:
Is this by design? we seldom downgrade service version.

Reply:
Just for testing.